test_runner/regress/test_disk_usage_eviction.py

```diff
@@ -64,6 +65,26 @@ def assert_overrides(tenant_id, default_tenant_conf_value):
     assert_config(tenant_id, None, config_level_override)
 
 
+@enum.unique
+class EvictionOrder(str, enum.Enum):
+    ABSOLUTE_ORDER = "absolute"
+    RELATIVE_ORDER_EQUAL = "relative_equal"
+    RELATIVE_ORDER_SPARE = "relative_spare"
+
+    def as_toml(self):
```

Comment:
should this be annotated with a return type?

Reply:
They'll go to encoders so I did not want but since you asked...

Reply:
3b81ab2a0